src/site/content/en/blog/native-lazy-loading/index.md

```diff
@@ -114,22 +114,14 @@ to add `height` and `width` attributes to the `<img>` element or specify their v
 inline style:
 
 ```html
-<img src="..." loading="lazy" width="200" height="200">
-<img src="..." loading="lazy" style="height:200px; width:200px;">
-```
-
-Images will still lazy-load if dimensions are not included, but specifying them decreases the chance of
-browser reflow.
-
-Support for the `intrinsicsize` attribute is also being [worked
-on](https://bugs.chromium.org/p/chromium/issues/detail?id=967992), so images will lazy-load
-correctly if `intrinsicsize` is specified along with one other dimension (`width` or `height`).
-
-```html
-<img src="…" alt="…" loading="lazy" intrinsicsize="250x200" width="450">
+<img src="..." loading="lazy" alt="..." width="200" height="200">
```

Comment:
```suggestion
<img src="…" loading="lazy" alt="…" width="200" height="200">
```

Just fixing something that should have been caught in earlier reviews. We use the ellipsis character `…` rather than 3 period characters `...` because screen readers announce the first as "ellipsis" whereas the second is announced as "period period period"